packages/web/src/pages/search/search.tsx

```diff
@@ -96,6 +102,76 @@ const userCard = (
   );
 };
 
+const intelBulletin = (
+  name: string,
+  descriptionShort: string,
+  icon: string,
+  serverID: string,
+  push: {
+    (path: string, state?: unknown): void;
+    (location: History.LocationDescriptor<unknown>): void;
+    (arg0: string): void;
+  },
+) => {
+  const iconBulletin = getBulletinIconPath(icon);
+  const nameBulletin = name;
+  const descriptionBulletin = descriptionShort;
+
+  return (
```

Comment:
What we could do on click on the bulletin, we could redirect to search bulletin page like this.
So in case you search for `infantry` we could to this:
https://coh2stats.com/bulletins?search=infantry

You can get the `routes.bulletinsBase` the get the url. And you can build the search param like this https://stackoverflow.com/a/49701878/1768843